project/frontend/src/containers/Marketplace/Marketplace.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import classes from "./Marketplace.module.css";
+import Block from "../../components/UI/Block/Block";
+import TrendingSearches from "../../components/TrendingSearches/TrendingSearches";
+import RecentBuys from "../../components/RecentBuys/RecentBuysList";
+import Button from "../../components/UI/Button/Button";
+
+const Marketplace = (props) => {
+  console.log(props);
+  return (
+    <div className={classes.MarketplaceContainer}>
+      <div className={classes.BuyContainer}>
+        <div className={classes.SearchContainer}>
+          <input
+            className={classes.SearchInput}
+            placeholder="Google search here"
```

Comment:
Feels like the word Trend should be in the placeholder

Reply:
CHange d it to "Search a trend"